storage/s3.go

```diff
@@ -751,19 +772,3 @@ func errContains(err error, msg string) bool {
 func IsCancelationError(err error) bool {
 	return errHasCode(err, request.CanceledErrorCode)
 }
-
-// writerAtAdapter is an 'io.Writer' adapter for 'io.WriterAt'.
-type writeAtAdapter struct {
```

Comment:
Why this block is removed? Cat command uses this struct.

Reply:
it was being used only for one scenario with `storage.Get()` where is concurrency is 1, since we took that into a seperate function, this was not necessary to keep

Reply:
thanks. I overlooked `storage.Open()` implementation.